content/sensu-go/6.7/observability-pipeline/observe-schedule/agent.md

```diff
@@ -1373,6 +1382,22 @@ sensu-agent start --keepalive-interval 30{{< /code >}}
 /etc/sensu/agent.yml example | {{< code shell >}}
 keepalive-interval: 30{{< /code >}}
 
+<a id="keepalive-pipelines-flag"></a>
+
+| keepalive-pipelines |      |
+--------------------|------
+description         | [Pipelines][63] to use for processing keepalive events, specified in a comma-delimited list. If keepalive pipelines are not specified, the Sensu backend will use the default `keepalive` handler and create an event in sensuctl and the Sensu web UI.
```

Comment:
I'm not totally sure how I'd word it, but it's actually more like a comma-delimited list of resource reference stings rather than just pipeline names. Your examples would translate to `--keepalive-pipelines core/v2.Pipeline.slack,core/v2.Pipeline.email`

I think that this was the first configuration option that needed a full resource reference, so it's a new concept to configuration. We borrowed the idea of resource reference strings from sensuctl, so it should be pretty familiar. example: `sensuctl edit core/v2.Pipeline my_pipeline`.

Here's a handful of examples from our test cases: https://github.com/sensu/sensu-go/blob/3a31c256f42c47adcd918079c7980ff201d6487e/api/core/v2/resource_reference_test.go#L13-L38. You'll notice it supports both the sensuctl edit style "core/v2.Pipeline testcase" but also "core/v2.Pipeline.testcase" since spaces can be kind of confusing in cli flags.

Reply:
ahhh thank you. You even called that out in the PR! I'll fix it.